cpg_workflows/stages/gatk_sv/gatk_sv_multisample_1.py

```diff
@@ -46,14 +46,14 @@ def expected_outputs(self, cohort: Cohort) -> dict[str, Path]:
             'cnmops_large_del_index': 'DEL.large.bed.gz.tb',
             'cnmops_large_dup': 'DUP.large.bed.gz',
             'cnmops_large_dup_index': 'DUP.large.bed.gz.tbi',
-            'merged_SR': 'sr.txt.gz',
-            'merged_SR_index': 'sr.txt.gz.tbi',
-            'merged_PE': 'pe.txt.gz',
-            'merged_PE_index': 'pe.txt.gz.tbi',
-            'merged_BAF': 'baf.txt.gz',
-            'merged_BAF_index': 'baf.txt.gz.tbi',
-            'merged_bincov': 'RD.txt.gz',
-            'merged_bincov_index': 'RD.txt.gz.tbi',
+            'merged_SR': f'{self.name}.sr.txt.gz',
```

Comment:
I'm a bit late suggesting this 😄 but I'd suggest a comment at the top of this bunch mentioning that they need the `.sr.txt.gz` etc extensions. Hopefully help avoid us unfixing this again in future!